tmux-onedark-theme.tmux

```diff
@@ -12,6 +12,26 @@ get_widgets() {
    echo "$(tmux show-option -gqv "@onedark_widgets")"
 }
 
+get_variable() {
+  echo "$(tmux show-option -gqv "@onedark_$1")"
+}
+
+get_time_string() {
+  if [ "$(get_variable "twelve_hour")" == "on" ]; then
+    echo "%l:%M %p"
+  else
+    echo "%H:%M"
+  fi
+}
+
+get_date_string() {
+  if [ "$(get_variable "month_first")" == "on" ]; then
```

Comment:
I think that instead of a "month_first" variable we should support "date_format" instead.
If the date format wasn't supplied, fallback to `%m/%d/%y`, otherwise use the one supplied. thoughts?